### PR TITLE
Separate user sessions app

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@marykhan

File: user_sessions/apps.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class LoginConfig(AppConfig):
```
Comment (MaryKhan) [inline user_sessions/apps.py]: rename it to UserSessionsConfig.

File: user_sessions/urls.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from . import views
+
+
+app_name = 'user_sessions'
+urlpatterns = [
+    url(r'^$', views.HomePageView.as_view(), name='home'),
+    url(r'^user_sessions/?$', views.LoginView.as_view(), name='login'),
```
Comment (MaryKhan) [inline user_sessions/urls.py]: url should be `login` here.

File: user_sessions/views.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+class HomePageView(View):
+    template_name = 'phone_directory/home.html'
+
+    def get(self, request):
+        return render(request, self.template_name)
+
+
+class LogoutView(View):
+    template_name = 'phone_directory/home.html'
```
Comment (MaryKhan) [inline user_sessions/views.py]: this template is associated with HomePageView. we should not render it directly from here, instead redirect to home view after logout.

**Review by MaryKhan — CHANGES_REQUESTED** [review]

**Review by MaryKhan — APPROVED** [review]
good to merge!

**Pull request merged**